config.js

```diff
@@ -228,7 +228,7 @@ export default [
       {
         name: 'accessibility concern',
         color: 'D93F0B',
-        description: 'Any bug, feature request or question about the accessibility of a portion of a product',
+        description: 'Any bug, feature request or question about the accessibility of a portion of a product that isn\'t a WCAG failure',
```

Comment:
How about a slight wording change to make the logic a bit clearer?

```suggestion
        description: 'Any bug, feature request or question about the accessibility of a portion of a product, but is not a WCAG failure',
```